docs/tutorial.rst

```diff
@@ -3,6 +3,113 @@
+the species default parameters. This includes selecting the demographic model,
+chromosome, recombination map, and number of samples. 
+
+The most basic simulation we can run is to draw two samples using the species'
+defaults as seen in the species help (``stdpopsim homsap --help``). These
+defaults include constant size population, uniform recombination map, and a
+default chromosome. To save time we will also specify that the simulation use
+chromosome 22 with the ``-c`` option. 
+
+.. code-block:: console
+
+    $ stdpopsim homsap -c chr22 -o foo.ts 2
+
+.. warning:: It's important to remember to either redirect the output of ``stdpopsim``
+                to file or to use the ``-o/--output`` option. If you do not, the 
+                binary output may mess up your terminal session.
+
+Say we want to use a specific demographic model. We look up the available models
+using the ``--help-models`` flag:
+
+.. code-block:: console
+
+    $ stdpopsim homsap --help-models
+
+This gives all of the possible demographic models we could simulate. We choose
+the the two population out-of-Africa model from `Tennesen et al.
+<https://doi.org/10.1126/ science.1219240>`_ . By looking at the model help we
+find that the name for this model is ``ooa_2`` and that we can specify it using
+the ``--model`` option. We choose to draw two samples from the African
+population and three samples from the European population. To increase
+simulation speed we can also chose to simulate only a subset of the chromosome
+using the ``-l`` option (e.g. 5%). The command now looks like this:
+
+.. code-block:: console
+
+    $ stdpopsim homsap -c chr22 -l 0.05 -o foo.ts --model ooa_2 2 3
+
+Note that there are now two numbers after the model option. This is because the
+model simulates two populations so we have to specify a number of samples to
+draw from each population (the order of those numbers is the same as the order
+specified in the model documentation). In this case, we are simulating two
+African American samples and three European American samples.
+
+Now we want to add an empirical recombination map to make the simulation more
+realistic. We can run ``stdpopsim homsap --help-genetic-maps`` to view the
+available recombination maps. In this case we choose the HapmapII map. Empirical
+recombination maps cannot be used with length multipliers so we have to remove
+the ``-l`` option. (NOTE: this may a minute or so to run).
+
+.. code-block:: console
+
+    $ stdpopsim homsap -g HapmapII_GRCh37 -c chr22 -o foo.ts --model ooa_2 2 3
+
+For reproducibility we can also choose set seed for the simulator using the
+``-s`` flag.
+
+.. code-block:: console
+
+    $ stdpopsim homsap -s 1046 -g HapmapII_GRCh37 -c chr22 -o foo.ts --model ooa_2 2 3
+
+Lastly, the CLI also outputs the relevant citations for both the simulator used
+and the resources used for simulation scenario.
+
+*****************************
+Using ``stdpopsim`` in python
```

Comment:
As per a previous comment, I'd change this heading title to 'The Python interface'. Add a section reference above the title: `.. sec_python_tute:` so we can link to it.